modules/core/templates/frontpage_federation.tpl.php

```diff
@@ -10,9 +10,9 @@
         $this->t('{core:frontpage:login_as_admin}').'</a></p>';
 }
 
-$now = time();
-echo '<dl>';
 if (is_array($this->data['metaentries']['hosted']) && count($this->data['metaentries']['hosted']) > 0) {
+    $now = time();
```

Comment:
I think this breaks the current code because `$now` is not used inside this `if()` block but in the next one, so if this `if` is not true, then `$now` will be unset.